v2/passwordless/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -2,20 +2,29 @@
 id: embed-sign-in-and-up-form
 title: Embed Sign In / Up form in a page
 hide_title: true
+show_ui_switcher: true
 ---
-
-import FrontendSDKTabs from "/src/components/tabs/FrontendSDKTabs"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs"
 import TabItem from '@theme/TabItem';
 import {Question, Answer}from "/src/components/question"
 import RRDVersionSubTabs from "/src/components/tabs/RRDVersionSubTabs"
 
 # Embed Sign In / Up form in a page 
 
+<PreBuiltOrCustomUISwitcher>
+
+<PreBuiltUIContent>
+This guide is not applicable for Pre built UI.
+</PreBuiltUIContent>
```

Comment:
isn't it the other way around? This guide isn't applicable for users who use custom UI.

Reply:
Yeah.. I got confused. Fixed it now.